ui/job-view/JobView.jsx

```diff
@@ -162,6 +165,14 @@ class JobView extends React.Component {
     document.getElementById('favicon').href = thFavicons[status] || thFavicons.open;
   }
 
+  // If showing is a boolean, then set to that value.  If it's not, then toggle
+  showOnScreenShortcuts(showing) {
```

Comment:
tbh, I wasn't quite sure what the right name for this function is.  I could have stuck with ``toggleOnScreenShortcuts``.  Happy to accept opinions.  :)

Reply:
The `showOnScreenShortcuts` name sounds great :-)